pkg/webhook/webhook.go

```diff
@@ -832,3 +892,43 @@ func SetInjectHugepageDownApi(hugepageFlag bool) {
 func SetHonorExistingResources(resourcesHonorFlag bool) {
 	honorExistingResources = resourcesHonorFlag
 }
+
+// SetCustomizedInjections sets additional injections to be applied in Pod spec
+func SetCustomizedInjections(injections *corev1.ConfigMap) {
+	var patch jsonPatchOperation
+	var cusPatchs = cusInjects.Patchs
+
+	for k, v := range injections.Data {
+		existValue, exists := cusPatchs[k]
+		// unmarshall customized injection to json patch
+		err := json.Unmarshal([]byte(v), &patch)
+		if err != nil {
+			glog.Errorf("Failed to unmarshall customized injection: %v", v)
+			continue
+		}
+		// metadata.Annotations is the only supported field for customization
+		// jsonPatchOperation.Path should be "/metadata/annotations" when customizing pod annotation
+		if patch.Path != "/metadata/annotations" {
+			glog.Errorf("Path: %v is not supported, only /metadata/annotations can be customized", patch.Path)
```

Comment:
The limitation we impose on the path should be mentioned in the README

Reply:
Done